mod_wsbridge/mod_wsbridge.c

```diff
@@ -101,6 +104,41 @@ switch_endpoint_interface_t *wsbridge_endpoint_interface;
 static switch_memory_pool_t *module_pool = NULL;
 static int running = 1;
 
+struct Queue {
+	switch_queue_t *queue;
+	switch_mutex_t *mutex;
+	unsigned int count;
+	unsigned int size;
+};
```

Comment:
I think we can remove `count` and `size`.  We don't need them because switch_queue_size() tracks the number of elements in the queue.